.hgignore

```diff
@@ -0,0 +1,7 @@
+build
```

Comment:
Can we remove this file? The main repository is a git repository not a mercurial (AFAIK).